project4/project/ebs/templates/ebs/update_blog.html

```diff
@@ -0,0 +1,113 @@
+{% extends "ebs/common.html" %}
+{% load static %}
+{% block head_extra %}	
+<link rel="stylesheet" href="http://code.jquery.com/ui/1.11.0/themes/smoothness/jquery-ui.css">
+<link rel="stylesheet" type="text/css" href="{% static 'ebs/css/add-blog.css' %}"/>
+<link rel="stylesheet" type="text/css" href="{% static 'ebs/css/edit_blog.css' %}"/>
+<link rel="stylesheet" type="text/css" href="{% static 'ebs/css/bootstrap-datetimepicker.min.css' %}"/>
+{{ blogform.media }}
+{% endblock %}
+{% block content %}
+<div class="container">
+    <h3 class="header">EDIT BLOG</h3>
+    <form id="createform" method="post" enctype="multipart/form-data" action="{% url 'update_blog' id=bloginstance.id %}">
+        {% csrf_token %}
+        <div class="row">
+            <div class="form-group col-md-12">
+                {{blogform.title.errors}}
+                <label for="title" class="control-label">*Title</label>
+                {{blogform.title}}
+            </div>
+        </div>
+        <div class="row">
+            <div class="form-group col-md-12">
+                {{blogform.description.errors}}
+                <label for="description" class="control-label">*Description</label>
+                {{blogform.description}}
+            </div>
+        </div>
+        <div class="row">
+            <div class="form-group col-md-12">
+                {{blogform.tags.errors}}
+                <label for="tags" class="control-label">Tags</label>
+                {{blogform.tags}}
+            </div>
+        </div>
+        <div class="row">
+            <div class="form-group col-md-6">
+                {{blogform.image.errors}}
+                <label class="control-label">Attachments</label><br>
+                    {{blogfileform.attachments}}
+                    {{blogfileform.image1}}
+                    {{blogfileform.image2}}
+            </div>
+        </div>
+        <div class="row">
+            <div class="upload col-md-6">
+        {% if attachments_value %}
+                <label id="label-uploaded">Uploaded Files</label>
```

Comment:
- Change the layout of uploaded files list as suggested.
 - Keep the delete button aligned to right side.